Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -120,6 +120,7 @@ typedef NS_OPTIONS(NSUInteger, _WKRectEdge) {
 @class WKBrowsingContextHandle;
 @class WKDownload;
 @class WKFrameInfo;
+@class _UIFindInteraction;
```

Comment:
Nit: Can we move this up underneath `@class UIEventAttribution;`?

Reply:
We typically sort everything in ASCII order, respecting underscores and such.

Reply:
In that case, we should move this after `@class WKWebpagePreferences;`.